src/metabase/email/persisted-model-error.mustache

```diff
@@ -0,0 +1,111 @@
+{{> metabase/email/_header }}
+  {{! <style>
+    @media only screen and (max-device-width: 480px) {
+      html, body {
+        font-size: 20px;
+      }
+    }
+  </style> }}
+  <div class="card-container">
+    <h2
+      style="margin: 0; padding: 0; font-weight: 700; font-size: 1.5em; color: #4C5773; line-height: 2em;"
+    >
+      Model refresh for {{database-name}} failed
+    </h2>
+    {{#errors}}
+    <div class="padded-section">
+      <p
+      style="font-weight: 600; font-size: 0.875em; line-height: 1.375em; color: #949AAB;"
```

Comment:
nit: `{{colorTextMedium}}`

Reply:
also, seems like outlook has [weird behavior with line-height](https://www.caniemail.com/search/?s=line-height) that requires adding `mso-line-height-rule:exactly` to fix